modules/nf-core/deepvariant/makeexamples/main.nf

```diff
@@ -0,0 +1,64 @@
+process DEEPVARIANT_MAKEEXAMPLES {
+    tag "$meta.id"
+    label 'process_high'
+
+    //Conda is not supported at the moment
+    container "nf-core/deepvariant:1.6.1"
+
+    input:
+    tuple val(meta), path(input), path(index), path(intervals)
+    tuple val(meta2), path(fasta)
+    tuple val(meta3), path(fai)
+    tuple val(meta4), path(gzi)
```

Comment:
> Is this tuple val(meta4), path(gzi) needed?

TBC

Reply:
Think you can disregard my comment on this.

Reply:
Ok I will leave the option for gzi to make it easy :)  I didn't get a chance to test whether it works.